app/views/components/table_row.scala.html

```diff
@@ -16,16 +16,23 @@
 
 @import models.viewmodels.HtmlTableRow
 
-@(tableRow: HtmlTableRow)
+@(id: String, tableRow: HtmlTableRow)
 
 <tr class="section">
-    <td class="previous-question-title bold">@tableRow.label</td>
-    <td class="previous-question-body">
-    @for(value <- tableRow.values){
-        @value.getOrElse("None")<br/>
-    }
-    </td>
-    <td class="link-right">
-        <a href="">Change</a>
+    <td class="bold"><span id="@id-label">@tableRow.label</span></td>
+    <td>
+        @if(tableRow.values.drop(1).isEmpty) {
```

Comment:
Is this "if tableRow has one value"?  If so, why the extra logic?  Is the only difference the lack of an index number in the id?  Why not be consistent?

Reply:
yes, it's mean if tableRow has one element. `drop` is safer here than tail, because will return empty seq is the sequence will be empty. Tail will throw an exception. If I have one element or don't have any value here I don't want to have index in id

Reply:
why `drop(1).isEmpty` and not `size == whatever`

Reply:
Size is not recommended method to check isEmpty or nonEmpty without checking specific size, because it must iterate through whole sequence to find how many elements you have. Moreover in this case I'll need to check isEmpty and size == 1, so two cases

Reply:
All this interesting discussion doesn't explain what's wrong with "id=item-0" if there happens to be 1 item in the list?  That just means that whoever is writing the test(s) has to know/care how many items were there.

Reply:
It's table row not item. So this can be applied for additional document, package information and every place where we use it. Previous implementation was similar probably.

Doesn't matter for me how we want to deal with it. I can remove this part. For me it's better to have `additional-document` instead of `additional-document-0` if I have just one element

Reply:
Up to you.  I think `additional-document-0` is better and it makes the logic simpler.   (Both for view and test)

Reply:
As we discussed, if we change it here we will have `-0` in every id for every field. I'll try to find some more elegant solution

Reply:
We're using `HtmlTableRow` for it. It's difficult to achieve it without passing additional arguments or changing whole approach around this class